app-config.yaml

```diff
@@ -145,16 +157,16 @@ integrations:
     # - host: ghe.example.net
     #   rawBaseUrl: https://ghe.example.net/raw
     #   token: ${GHE_TOKEN}
-  gitlab:
-    - host: gitlab.com
-      token: ${GITLAB_TOKEN}
-  bitbucket:
-    - host: bitbucket.org
-      username: ${BITBUCKET_USERNAME}
-      appPassword: ${BITBUCKET_APP_PASSWORD}
-  azure:
-    - host: dev.azure.com
-      token: ${AZURE_TOKEN}
+  # gitlab:
+  #   - host: gitlab.com
+  #     token: ${GITLAB_TOKEN}
+  # bitbucket:
+  #   - host: bitbucket.org
+  #     username: ${BITBUCKET_USERNAME}
+  #     appPassword: ${BITBUCKET_APP_PASSWORD}
+  # azure:
+  #   - host: dev.azure.com
+  #     token: ${AZURE_TOKEN}
```

Comment:
Maybe when we've figured it out in the end, we can just remove what we don't need.